sklearn/calibration.py

```diff
@@ -242,6 +245,15 @@ class CalibratedClassifierCV(ClassifierMixin, MetaEstimatorMixin, BaseEstimator)
     array([[0.936..., 0.063...]])
     """
 
+    _parameter_constraints = {
+        "estimator": [BaseEstimator, None],
+        "method": [StrOptions({"isotonic", "sigmoid"}), None],
+        "cv": ["cv_object", StrOptions({"prefit"}), None],
```

Comment:
None is already included in `"cv_object"`.

```suggestion
        "cv": ["cv_object", StrOptions({"prefit"})],
```

Reply:
Hi, I will make these changes later today.